config/dev.env

```diff
@@ -4,7 +4,7 @@ AWS_S3_ENDPOINT=
 AWS_S3_EPUB_BUCKET=simplye-research-epubs
 
 AWS_KINESIS_ENDPOINT=
-AWS_KINESIS_STREAMNAME=sfr-epub-results-development
+AWS_KINESIS_STREAMNAME=sfr-db-ingest
```

Comment:
we need to have the '-development' string applied to stream names as per our standards.